storage/storage_test.go

```diff
@@ -30,14 +30,12 @@ import (
 	ddigest "github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
 )
 
 var (
-	_imgd      types.ImageDestination = &storageImageDestination{}
-	_imgs      types.ImageSource      = &storageImageSource{}
-	_ref       types.ImageReference   = &storageReference{}
-	_transport types.ImageTransport   = &storageTransport{}
```

Comment:
These things are actually tests that the various structs implement the necessary interfaces. We should find a way to preserve that.

Reply:
Unfortunately, I can't find the usage

Reply:
No “usage” is necessary. The lines are effective tests as they are (if `storage…` does not implement the interface, this will report an error).

The lines should be restored, and the warning, if any, silenced using a comment or whatever golang-CI uses.

Reply:
I can take care of that.

Reply:
Done (will open a PR in a couple of mins)

Reply:
Fixed in https://github.com/containers/image/pull/762 , for the record.